tests/test__default.py

```diff
@@ -637,6 +640,18 @@ def test__get_gcloud_sdk_credentials_no_project_id(load, unused_isfile, get_proj
     assert get_project_id.called
 
 
+def test__get_gdch_service_account_credentials_no_format_version():
+    with pytest.raises(exceptions.DefaultCredentialsError) as excinfo:
+        _default._get_gdch_service_account_credentials({})
+    assert excinfo.match("format_version is not provided or unsupported")
```

Comment:
shouldn't the error be "format_version is not provided or unsupported. Supported version is: v1"? Same below